.github/workflows/ci.yml

```diff
@@ -112,3 +112,13 @@ jobs:
         uses: github/codeql-action/upload-sarif@codeql-bundle-20211208
         with:
           sarif_file: trivy-results.sarif
+
+      - name: Upload binaries to release
+        if: ${{ github.ref == 'refs/heads/develop' || github.ref == 'refs/heads/master' }}
+        uses: svenstaro/upload-release-action@v2
+        with:
+          repo_token: ${{ secrets.GITHUB_TOKEN }}
+          file: target/connector.war
+          asset_name: ${{matrix.project_context}}_connector.war
```

Comment:
Maybe add %tag here to have a version number in the filename.

Reply:
Fixed with github.tag